reporter.go

```diff
@@ -70,41 +45,48 @@ func reportMetrics(ctx context.Context) {
 		}
 
 		if splitArn[5] == "function" {
-			hostTags["Resource"] = splitArn[6]
+			pointTags["Resource"] = splitArn[6]
 			if len(splitArn) == 8 {
-				hostTags["Resource"] += ":" + splitArn[7]
+				pointTags["Resource"] += ":" + splitArn[7]
 			}
 		} else if splitArn[5] == "event-source-mappings" {
-			hostTags["EventSourceMappings"] = splitArn[6]
+			pointTags["EventSourceMappings"] = splitArn[6]
+		}
+
+		// Send metrics using a Direct Wavefront Sender
+		err := sender.SendMetric("aws.lambda.wf.coldstarts", *csCounter, reportTime, lambdacontext.FunctionName, pointTags)
```

Comment:
Use sender.SendDeltaCounter instead. 
https://github.com/wavefrontHQ/wavefront-lambda-go#standard-metrics (cold starts, invocations and errors should be sent as delta counters)